cpp/contracts/feesmanager.cpp

```diff
@@ -0,0 +1,70 @@
+#include "feesmanager.hpp"
+
+using namespace eosio;
+
+name owner;
+
+void feesmanager::init( name security_council ) {
```

Comment:
Just wondering, is this space before/after parenthesis in the function prototype a thing in the usual cpp style guide? Because it makes my eyes bleeding... just asking :sweat_smile: